src/lib/cli_lib/port.ml

```diff
@@ -4,4 +4,6 @@ let default_client = 8301
 
 let default_external = 8302
 
+let default_rest = 0xc0da
```

Comment:
As much as I love `0xc0da` as a port number, it unfortunately lies inside the "Dynamic Port" range (https://tools.ietf.org/html/rfc6335) and is prone to being automatically used for ephemeral connections. With the number of TCP connections I see during casual use of my laptop, I guesstimate about a .8% chance binding this port will fail at any time. We should pick a port in the User Range (1024-49151).

Reply:
Haha good to know, we had been casually using this port for testing but this makes a lot of sense so I'll get that changed

Reply:
we could do `0x30da`? or `0x50da`